docs/linux/reporting_kernel_bugs.md

```diff
@@ -52,61 +52,119 @@ If you can't figure out the right fix, but have some understanding of the bug, p
-* The moment the issue is made public (e.g. patch is submitted upstream, CVE description published, etc.) it must be reported to `oss-security@lists.openwall.com` right away.
+As of [February 13, 2024](https://www.cve.org/Media/News/item/news/2024/02/13/kernel-org-Added-as-CNA), kernel.org is now CVE Numbering Authority (CNA).
+therefore, they have reserved CVE IDs for the kernel vulnerability bugs classified as CVE.
 
-A good example of an LPE announcement structure on `oss-security@lists.openwall.com` can be found [here](http://seclists.org/oss-sec/2016/q4/607), however the timeline doesn't look right there: public announcement should have occurred right after the patch was submitted to netdev.
+CVE assignments (sometimes rejections) are publicly announced in the public list, see [linux-cve-announce](https://lore.kernel.org/linux-cve-announce) on which you can [subscribe](https://subspace.kernel.org/subscribing.html).
+
+CVEs are assigned after-the-fact with 1-2 week delay usually.
+> No hardware CVEs (SPECTRE-like CVEs come from others). (c) Greg K-H
+
+, e.g. bugs in drivers are Linux kernel related, hence they can be treated as CVEs, and HW bugs in CPU, NIC, other chips that affect Linux kernel, Windows, OSX etc.
+that are not caused due to Linux kernel code, should be handled by HW vendor.
+
+Averaging 55 CVEs per week (2024 Q3):
+> The CVE assignment team is very thorough and assigns CVE numbers to any bugfix that they identify.
+> This explains the seemingly large number of CVEs that are issued by the Linux kernel team.
+
+Kernel CVE assignment process is explained [here](https://www.kernel.org/doc/html/latest/process/cve.html), quoting the important paragraph:
+> If the CVE assignment team misses a specific fix that any user feels should have a CVE assigned to it,
+> please email them at <cve@kernel.org> and the team there will work with you on it.
+> Note that no potential security issues should be sent to this alias,
+> it is ONLY for assignment of CVEs for fixes that are already in released kernel trees.
+> If you feel you have found an unfixed security issue, please follow the [normal Linux kernel security bug reporting process](https://docs.kernel.org/process/security-bugs.html).
+
+Note that kernel.org assigns CVE in a very descriptive format (which files, versions are affected etc.).
+See [CVE-2024-50078](https://lore.kernel.org/linux-cve-announce/2024102936-CVE-2024-50078-cfe9@gregkh/T/#u) as an example.
+
+### Process
+
+![reporting kernel bugs](./reporting_kernel_bugs.png)
+
+1. Understand the bug and develop a patch with a fix if possible. Optionally develop a proof-of-concept exploit.
+    * Notify `security@kernel.org`
+    * As the recommendation, do not include `linux-distros@vs.openwall.org` in conversation at this moment.
```

Comment:
It would be nice to add a link to the recommendation source.

Reply:
Added the link of recommendation from "Coordinating with other groups". Here is the sentence highlight:
https://docs.kernel.org/process/security-bugs.html#coordination-with-other-groups:~:text=the%20kernel%20security%20team%20strongly%20recommends%20that%20as%20a%20reporter%20of%20a%20potential%20security%20issue%20you%20DO%20NOT%20contact%20the%20%E2%80%9Clinux%2Ddistros%E2%80%9D%20mailing%20list%20UNTIL%20a%20fix%20is